libs/zflecs/build.zig

```diff
@@ -13,6 +13,7 @@ pub fn build(b: *std.Build) void {
         .target = target,
         .optimize = optimize,
     });
+    b.installArtifact(flecs);
```

Comment:
I'm not  sure what's going on with the diff, but `installArtifact(flecs)` is already being called on main branch, albeit not on this exact line. Can you try rebasing this branch?

Reply:
yes, will do.